subprojects/shipkit/shipkit.gradle

```diff
@@ -22,8 +22,9 @@ dependencies {
 
     compile "com.github.cliftonlabs:json-simple:2.1.2"
     compile 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-    compile "com.gradle.publish:plugin-publish-plugin:0.9.10"
+    compile 'com.gradle.publish:plugin-publish-plugin:0.9.10'
```

Comment:
Is it still needed?

Reply:
Gradle plugin itself seems to be needed since shipkit is also published to gradle plugin portal. However, change in this line is not needed.